packages/ui/react-components/package.json

```diff
@@ -22,17 +22,25 @@
         "nav-frontend-core": "^5.1.1",
         "nav-frontend-etiketter": "^2.1.1",
         "nav-frontend-etiketter-style": "^1.1.1",
+        "nav-frontend-knapper": "^2.0.32",
+        "nav-frontend-knapper-style": "^1.0.17",
         "nav-frontend-typografi": "^3.1.1",
         "nav-frontend-typografi-style": "^1.1.1",
         "react": "^17.0.2",
         "react-dom": "^17.0.2"
     },
     "peerDependencies": {
+        "classnames": "^2.3.1",
```

Comment:
Alt under `peerDependencies` her bør også være med i `devDependencies` (dette for at ting som Storybook skal fungere)